hcn/hcnnetwork.go

```diff
@@ -415,15 +415,22 @@ func GetNetworkByID(networkID string) (*HostComputeNetwork, error) {
 		return nil, err
 	}
 	hcnQuery.Filter = string(filter)
+	queryJson, err := json.Marshal(hcnQuery)
```

Comment:
>err [](http://example.com/codeflow?start=12&length=3)

is this possible to occur?

Reply:
i guess not. but keeping it in line with current implementation.

Reply:
i think we should leave as is. let me know if you prefer to ignore.